api/resource.rb

```diff
@@ -322,6 +324,17 @@ def check_identity
       end
     end
 
+    def check_url
```

Comment:
Can you add a comment describing what the "check" is?

Reply:
Done. I changed the function name too because that name is too vague.